widgets/MongoDB-batchsink.json

```diff
@@ -5,19 +5,77 @@
   "display-name": "MongoDB",
   "configuration-groups": [
     {
-      "label": "MongoDB Configuration",
+      "label": "Basic",
       "properties": [
         {
           "widget-type": "textbox",
           "label": "Reference Name",
-          "name": "referenceName"
+          "name": "referenceName",
+          "widget-attributes": {
+            "placeholder": "Name used to identify this sink for lineage"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Host",
+          "name": "host",
+          "widget-attributes": {
+            "default": "localhost"
```

Comment:
don't need to default the host, since we expect most users to replace this

Reply:
Removed.